packages/jsdoc-plugins/lib/relation-fixer/filteroutintenraldoclets.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+// @ts-check
+
+'use strict';
+
+/**
+ * Filters out doclet that won't be displayed.
+ *
+ * @param {Array.<Doclet>} doclets
+ */
+module.exports = function filterOutInternalDoclets( doclets ) {
+	return doclets
+		.filter( doclet => !doclet.ignore )
+		.filter( doclet => doclet.memberof != '<anonymous>' )
+		.filter( doclet => filterOutReExportedSymbols( doclet, doclets ) );
+};
+
+/**
+ * @param {Doclet} doclet
+ * @param {Array.<Doclet>} doclets
+ */
+function filterOutReExportedSymbols( doclet, doclets ) {
+	// Filter out exported symbols that are defined previously (JSDoc creates doclets for both).
+	const isUnwanted = ( doclet.scope == 'inner' && doclet.kind == 'constant' && doclet.undocumented );
```

Comment:
Useless `(` and `)`.

Reply:
It's a common practice to indicate that the value is a boolean AFAIK.

Reply:
Not in JS or at least not in our code style:

- https://github.com/ckeditor/ckeditor5-list/blob/a3122af2cfa493b0bb89d8737e24bc504fcf1071/src/listediting.js#L148
- https://github.com/ckeditor/ckeditor5-typing/blob/e553afb1592c66a45f0964ccb41d6f1672ad3702/src/utils/injectunsafekeystrokeshandling.js#L48
- https://github.com/ckeditor/ckeditor5-typing/blob/e553afb1592c66a45f0964ccb41d6f1672ad3702/src/deletecommand.js#L151
- https://github.com/ckeditor/ckeditor5-list/blob/e3b3ff3856077ee07cbef2ec14df9e3a8150fd2b/src/converters.js#L352